TI-reports/2025/2025-Q2-CTI.md

```diff
@@ -0,0 +1,70 @@
+
+Three documents were published in CY25Q2 (May) as work output from the CTI TAC and project developers:
+
+ * [GNU Toolchain SSDLC](https://sourceware.org/glibc/wiki/SSDLC)
+ * [GNU C Library SSDLC](https://sourceware.org/glibc/wiki/SSDLC/Policy/glibc)
+ * [Comparison of Sourceware Cyber Security project policy checklist](https://sourceware.org/glibc/wiki/SSDLC/Policy/Sourceware)
+
+The goal of these documents is to help the developers and volunteers understand the need for secure infrastructure and the best frameworks to approach these tasks.
+
+## Core Toolchain Infrastructure Project
+
+Core Toolchain Infrastructure Project (CTI) stalled in 2024 as the upstream developers raised questions around upstream secure development practices and why we might carry out these activities and how they support the projects.
+
+In order to engage the developers and discuss security and secure development practices it was decided that the CTI TAC would work with project developers on providing a set of guiding documents that defined a secure software development life-cycle (SSDLC) for the projects
+
+Three documents were published in CY25Q2 (May) as work output from the CTI TAC and project developers:
+
+ * [GNU Toolchain SSDLC](https://sourceware.org/glibc/wiki/SSDLC)
+ * [GNU C Library SSDLC](https://sourceware.org/glibc/wiki/SSDLC/Policy/glibc)
+ * [Comparison of Sourceware Cyber Security project policy checklist](https://sourceware.org/glibc/wiki/SSDLC/Policy/Sourceware)
+
+The goal of these documents is to help the developers and volunteers understand the need for secure infrastructure and the best frameworks to approach these tasks.
+
+### Purpose
+
+The Core Toolchain Infrastructure (CTI) Project’s mission is to support the GNU Toolchain community with secure infrastructure and state of the art services required to support the community’s development efforts to be a trusted foundation in a secure supply chain.
+
+The CTI project continues to move forward the goal of creating a long-term sustainable set of secure and state of the art services and infrastructure for the GNU Toolchain and related packages.
+
+Some of the major goals include:
+
+ * Secure and state of the art infrastructure.
+ * Continuity planning for infrastructure, development, and governance.
+ * Security policy planning.
+
+### MVSR
+
+Mission:
+ * To support the GNU Toolchain community with secure infrastructure and state of the art services required to support the community’s development efforts to be a trusted foundation in a secure supply chain.
+Vision:
+ * A thriving ecosystem of software that can be developed with the GNU Toolchain as a trusted foundation in a secure supply chain.
+Strategy:
+ * Review, prioritize, and support key infrastructure requirements from the GNU Toolchain community in collaboration with project sponsors.
+Roadmap
+ * Establish a baseline of secured services with support from project sponsors.
```

Comment:
The line breaks are broken in the rendered document. This may fix it. 

```suggestion
Mission:
 * To support the GNU Toolchain community with secure infrastructure and state of the art services required to support the community’s development efforts to be a trusted foundation in a secure supply chain.
 
Vision:
 * A thriving ecosystem of software that can be developed with the GNU Toolchain as a trusted foundation in a secure supply chain.
 
Strategy:
 * Review, prioritize, and support key infrastructure requirements from the GNU Toolchain community in collaboration with project sponsors.
 
Roadmap
 * Establish a baseline of secured services with support from project sponsors.
```